java/main/src/test/java/com/saucelabs/saucebindings/integration/AccessibilityTest.java

```diff
@@ -0,0 +1,68 @@
+package com.saucelabs.saucebindings.integration;
+
+import com.deque.html.axecore.results.Results;
+import com.deque.html.axecore.results.Rule;
+import com.saucelabs.saucebindings.SauceSession;
+import org.junit.After;
+import org.junit.Test;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class AccessibilityTest {
+    private SauceSession session = new SauceSession();
+    private RemoteWebDriver webDriver;
+
+    @After
+    public void cleanUp() {
+        if (session != null) {
+            session.stop(true);
+        }
+    }
+
+    @Test
+    public void analyzesPageWithNestedIFrames() {
+        webDriver = session.start();
+        webDriver.get("http://watir.com/examples/nested_iframes.html");
+
+        Results results = session.accessibilityResults();
+        List<Rule> violations = results.getViolations();
+        int problems = violations.stream().map(e -> e.getNodes().size()).reduce(0, Integer::sum);
+        assertEquals(16, problems);
```

Comment:
Again, new to automated accessibility testing. If this is the typical usage and users are validating that no violations exist, does it make sense to provide a simpler method for users:
`assertEquals(0, results.getViolationsCount())` which will do `violations.stream().map(e -> e.getNodes().size()).reduce(0, Integer::sum)` inside of `getViolationsCount()`

Reply:
The functionality is mostly just to populate the tab in the UI. There are better ways to test the code is working as intended, this was the "easy" way for me to make sure it was capturing exactly what I expected w/ and w/o frames. If users are interested in a more convenient Results class, we can add one. I just didn't want to add it from the get go without knowing if people were going to be using it or if they cared.